lib/core/text/abstract_text.nit

```diff
@@ -602,7 +605,13 @@ abstract class Text
 	#     assert "{0.code_point}{1.code_point}{8.code_point}{31.code_point}{32.code_point}".escape_to_c == "\\000\\001\\010\\037 "
 	#
 	# The exceptions are the common `\t` and `\n`.
-	fun escape_to_c: String
+	fun escape_to_c: String do return escape_to_c_text_impl
+
+	# The original definition of `escape_to_c`
+	#
+	# This method is separated from `escape_to_c` mainly for testing purposes.
+	# Normally, you should not call this method explicitly nor redefine it.
+	protected fun escape_to_c_text_impl: String
```

Comment:
Honestly I'm not a fan of having a new protected method just to add a test suite, would it be possible not to change the methods within text for testing and just have the contents of `text_abstract_text` be simple nitunits?